Balance/Models/EthereumWallet.swift

```diff
@@ -13,6 +13,16 @@ struct EthereumWallet {
     var currency: String?    // Fiat currency used for price and balance
     
     var tokens: [Token]?
+    var nonZeroTokens: [Token]? {
+        get {
+            return tokens?.filter({
```

Comment:
We don't have an official style guide posted, but trying to keep things consistent. All blocks that can be done this way (i.e. not inside an if statement where it's clearer to use parenthesis) should have this format:

```
return tokens?.filter {
    if let fiat = $0.fiatBalance {
        return fiat >= 10.0
    }
    return false
}
```

Reply:
sorted

Reply:
oh actually this can be compacted a lot I think. Something like:

`return tokens?.filter { $0.fiatBalance ?? 0 >= 10.0 }`

Reply:
don't even know how this works but it does

Reply:
@wolffan basically it just gives the optional value a default so that it's no longer optional. Because you can't do greater than comparisons on optionals against constants (or even any non-optional I think).

So `$0.fiatBalance ?? 0` is just saying "if fiatBalance is nil, use 0 instead" so now fiatBalance is always a number and the comparison works. Just a little annoying thing about optionals.